### PR TITLE
Add support for SLES12SP3 repositories

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

**Comment by jsuchome** [comment]
I'm actually not sure about the values in `repos-cloud.yml` - where do I find correct fingerprints?

File: crowbar_framework/lib/crowbar/product.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ def self.is_ses?
 
     def self.ses_platform
       # Returns the platform used for SES
-      "suse-12.2"
+      "suse-12.3"
```
Comment (dirkmueller) [inline crowbar_framework/lib/crowbar/product.rb]: do we want to switch to ses5 right away? I would expect this to have further fallout..

Comment (jsuchome) [inline crowbar_framework/lib/crowbar/product.rb]: Does SES4 work with SP3?

**Comment by jsuchome** [comment]
Maybe I need to remove all those `suse-12.2` repos as well..

**Comment by dirkmueller** [comment]
@jsuchome no this is just because gating required the crowbar PR to be merged first. thats why I suggested the automation workaround which would have solved that as well ;)

**Comment by jsuchome** [comment]
[#cloud8 Check there's no cookbook code that is SP2-specific #sp3](https://trello.com/c/TqH64Qo0/28-cloud8-check-theres-no-cookbook-code-that-is-sp2-specific-sp3)

File: crowbar_framework/lib/crowbar/product.rb (hunk 1 above)
Comment (dirkmueller) [inline crowbar_framework/lib/crowbar/product.rb]: no, its sp2, it means mkcloud needs to deploy ceph nodes with sp2 then (I think we used to have code for that)

File: crowbar_framework/config/repos-cloud.yml [hunk 2]
```diff
@@ -276,3 +276,272 @@ suse-12.2:
...
+      repomd:
+        tag: ["obsproduct://build.suse.de/SUSE:SLE-12-SP3:Update:Products:Cloud8/suse-openstack-cloud/8/cd/x86_64",
+              "obsproduct://build.suse.de/Devel:Cloud:8:Staging/suse-openstack-cloud/8/cd/x86_64",
+              "obsproduct://build.suse.de/Devel:Cloud:8/suse-openstack-cloud/8/cd/x86_64"]
+        fingerprint: []
+      url:
+      ask_on_error: false
+    suse-openstack-cloud-8-pool:
+      name: "SUSE-OpenStack-Cloud-8-Pool"
+      required: "mandatory"
```
Comment (dirkmueller) [inline crowbar_framework/config/repos-cloud.yml]: I think we need to make it optional until the scc/ibs setup is there..

Comment (jsuchome) [inline crowbar_framework/config/repos-cloud.yml]: Done

File: crowbar_framework/lib/crowbar/product.rb (hunk 1 above)
Comment (jsuchome) [inline crowbar_framework/lib/crowbar/product.rb]: I could revert this to "suse-12.2" and we can create separate card for SES deployment investigation, what do you think?

**Comment by jsuchome** [comment]
Travis failure is fixed with https://github.com/crowbar/crowbar-core/pull/1248

File: crowbar_framework/lib/crowbar/product.rb (hunk 1 above)
Comment (dirkmueller) [inline crowbar_framework/lib/crowbar/product.rb]: as said: lets try with sp3 and see how far we get

**Comment by dirkmueller** [comment]
it went far enough..

**Pull request merged**